doc/source/user_guide/fields/reduction.rst

```diff
@@ -321,7 +323,7 @@ Object-Oriented:
 .. code-block:: python
 
   >>> reduction.sum_if(
-  >>>   expression="AbsolutePressure",
+  >>>   expression=VariableCatalog.ABSOLUTE_PRESSURE,
   >>>   condition="AbsolutePressure > 0[Pa]",
   >>>   locations=solver_session.settings.setup.boundary_conditions.velocity_inlet,
   >>>   weight="Area"
```

Comment:
Can we have something for condition and weight as well using VariableCatalog? Any ideas? @seanpearsonuk, @mkundu1, @hpohekar